AnnoyancesFilter/sections/popups.txt

```diff
@@ -90,6 +90,8 @@ sodapdf.com##.be-tooltip
 !
 tube8.es###catFishBanner
 tube8.es###announcement_ticker
+hasepost.de#$#.pum-overlay { display: none!important; }
+hasepost.de#$#html { overflow: visible!important; }
```

Comment:
Changing it to .pum-open-overlay instead of html.

Reply:
this is ok